prijateli_tree/app/routers/games.py

```diff
@@ -34,16 +34,18 @@ def route_add_player(
 
 
 @router.get("/{game_id}")
-def route_game_access(game_id: int):
-    game = Game.query().filter_by(id=game_id).one_or_none()
+def route_game_access(game_id: int, db: Session = Depends(get_db)):
+    game = db.query(Game).filter_by(id=game_id).one_or_none()
```

Comment:
Why can't you do `Game.query().filter_by...`?

Reply:
It was not allowing me to do it! The only way I got it  to work was with the 'db' object